From: Andrew Morton <akpm@linux-foundation.org>
Date: Wed, 15 Apr 2026 22:47:13 -0700
Subject: [PATCH] maintainers-add-page-cache-reviewer-fix

add linux-mm@kvack.org

Cc: Jan Kara <jack@suse.cz>
Cc: Matthew Wilcox (Oracle) <willy@infradead.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---
 MAINTAINERS | 1 +
 1 file changed, 1 insertion(+)

diff --git a/MAINTAINERS b/MAINTAINERS
index de8c8f107eebd..ab54a9c77603e 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -19966,6 +19966,7 @@ PAGE CACHE
 M:	Matthew Wilcox (Oracle) <willy@infradead.org>
 R:	Jan Kara <jack@suse.cz>
 L:	linux-fsdevel@vger.kernel.org
+L:	linux-mm@kvack.org
 S:	Supported
 T:	git git://git.infradead.org/users/willy/pagecache.git
 F:	Documentation/filesystems/locking.rst
-- 
2.47.3

