From: Andrew Morton <akpm@linux-foundation.org>
Date: Thu, 23 Apr 2026 06:52:34 -0700
Subject: [PATCH] proc-rewrite-next_tgid-fix

remove newline

Cc: Alexey Dobriyan <adobriyan@gmail.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---
 fs/proc/base.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/fs/proc/base.c b/fs/proc/base.c
index f38ac943ec32a..49344de101582 100644
--- a/fs/proc/base.c
+++ b/fs/proc/base.c
@@ -3546,8 +3546,7 @@ struct tgid_iter {
 	struct pid_namespace *const pid_ns;
 };
 
-static
-struct tgid_iter
+static struct tgid_iter
 make_tgid_iter(unsigned int init_tgid, struct pid_namespace *pid_ns)
 {
 	return (struct tgid_iter){
-- 
2.47.3

